asistente_ladm_col/gui/supplies/dlg_missing_snc_supplies.py

```diff
@@ -74,10 +75,11 @@ def accepted(self):
         self.save_settings(self.data_system)
 
         self.folder_path = self.txt_file_path_folder_supplies.text()
+        self.files_name = self.txt_files_name_supplies.text().strip()
         self.gpkg_path = os.path.join(self.folder_path, QCoreApplication.translate(
-                'MissingSncSuppliesDialog', 'missing_supplies_snc.gpkg'))
+                'MissingSncSuppliesDialog', '{}.gpkg'.format(self.files_name)))
         self.xlsx_path = os.path.join(self.folder_path, QCoreApplication.translate(
-                'MissingSncSuppliesDialog', 'missing_supplies_snc.xlsx'))
+                'MissingSncSuppliesDialog', '{}.xlsx'.format(self.files_name)))
```

Comment:
No need for translations for a file name given by the user.

Reply:
Solved.